tsconfig-strict.json

```diff
@@ -31,12 +31,19 @@
     }
   },
   "files": [
+    "core/templates/domain/skill/assigned-skill-object.factory.ts",
+    "core/templates/domain/skill/assigned-skill-object.factory.spec.ts",
+    "core/templates/domain/utilities/AudioFileObjectFactory.ts",
```

Comment:
Please cover spec file for this too.

Reply:
hi @nishantwrp there is no spec file for the ```AudioFileObjectFactory.ts``` .

Reply:
Ah, okay.

Reply:
@nishantwrp should I ask the other authors as well for the pr or they will automatically see it. I don't know what I should do now.

Reply:
@arpit1912 as you can see the other reviewers are automatically assigned by the @oppiabot.

Reply:
Cool!, just after I commented the oppia bot did it.